grpc-protocol/src/main/java/com/linecorp/armeria/client/grpc/protocol/UnaryGrpcClient.java

```diff
@@ -72,11 +82,24 @@
     // cannot currently implement this without rethinking / refactoring core and punt for now since this is an
     // advanced API.
     public UnaryGrpcClient(WebClient webClient) {
+        this(webClient, UnaryGrpcSerializationFormats.PROTO);
+    }
+
+    /**
+     * Constructs a {@link UnaryGrpcClient} for the given {@link WebClient} and {@link SerializationFormat}.
```

Comment:
Could we explain what kinds of `SerializationFormat` are expected? For example:
```
The specified {@link SerializationFormat} should be one of {@link UnaryGrpcSerializationFormats.PROTO}, {@link UnaryGrpcSerializationFormats.PROTO_WEB} or {@link UnaryGrpcSerializationFormats.PROTO_WEB_TEXT}. 
```

Reply:
Sorry, we could link internal API to public API. Could you replace `@link` with `@code` for `UnaryGrpcSerializationFormats`? 😅